packages/react/src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { Children, useState, forwardRef, type Ref, useId } from 'react';
+import { Provider } from 'react-aria-components';
+import { cx } from 'cva';
+import { ChevronDown } from '@obosbbl/grunnmuren-icons-react';
+
+import { useClientLayoutEffect } from '../utils/useClientLayoutEffect';
+import { HeadingContext, ContentContext } from '../content';
+
+type AccordionProps = {
+  children: React.ReactNode;
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+};
+
+type AccordionItemProps = {
+  children?: React.ReactNode;
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+
+  /** Whether the accordion is open (controlled) */
+  isOpen?: boolean;
+  /** Whether the accordion is open by default (uncontrolled) */
+  defaultOpen?: boolean;
+  /** Handler that is called when the accordion's open state changes */
+  onOpenChange?: (isOpen: boolean) => void;
+};
+
+function Accordion(props: AccordionProps, ref: Ref<HTMLDivElement>) {
+  const { children, ...restProps } = props;
+
+  const childCount = Children.count(children);
+
+  return (
+    <div {...restProps} ref={ref}>
+      {Children.map(children, (child, index) => {
+        return (
+          <>
+            {child}
+            {index < childCount - 1 && <hr className="border-gray-light" />}
```

Comment:
Skjule `<hr/>` for skjermlesere her kanskje?

```suggestion
            {index < childCount - 1 && <hr className="border-gray-light" aria-hidden="true" />}
```